.github/workflows/mig.yml

```diff
@@ -0,0 +1,82 @@
+name: mig-test
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Switch to build on push to main:
```
on:
  push:
    branches: main
    tags:
      - v*
```

GitHub actions will not use your the repo secrets on pull request, so we will just need to merge it to test

Reply:
Sadly :(